projects/packages/search/src/instant-search/components/dom-event-handler.jsx

```diff
@@ -190,6 +191,14 @@ export default class DomEventHandler extends Component {
 		}
 	};
 
+	fixBodyOpacity = () => {
+		if ( '1' !== document.body.style.opacity ) {
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)

Would it be possible to change this to the following?

```suggestion
		if ( '0' === document.body.style.opacity ) {
```

I am a bit concerned that this executes unnecessarily, such as when `document.body.style.opacity` is unset.

Reply:
Changed https://github.com/Automattic/jetpack/pull/40472/commits/b923e01376b90b0c764984b6f21027d6c3c8badd